sdk/va-report-components/documentation/docs/api/SASReportObjectElement.md

```diff
@@ -62,6 +62,12 @@ default value: `true`
 default value: `undefined`<br>
 default behavior: use a shared report context per report
 
+### `hideLoadImage: boolean`
+
+Indicates whether the static loading image displayed during object load should be hidden. When true, the report image is not displayed.
```

Comment:
Can "static loading image" change at the beginning of the first sentence? What about this: "Indicates whether the static report image should be hidden when the object is loaded."

Reply:
I was looking at existing doc to see if we have precedence for wording and sentence structure for anything similar.  The closest I can see is the doc for [`restrictViewportGestrues`](https://developer.sas.com/sdk/va/docs/api/SASReportObjectElement/#restrictviewportgestures-boolean). 

It says
> When `true`, report objects that support zooming require a modifier...

So the parallel here might be
> When `false`, report objects display a placeholder static image of the report object while it is loading. `true` hides this placeholder and displays a loading indicator.
> 
> default value: `false`

Reply:
That looks good to me, @brmorr. Thanks for finding something similar that we already had.